### ISSUE TITLE
Accessibility: Contrast issues on home page

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#231]
Me again! I think I forgot to log these contrast issues on the home page. I think for the more part these are the last of the contrast issues.

- When you hover over each news/profile, the picture has a dark overlay and the words "view" with an arrow. the view/arrow combination is saying there is a foreground color of #fff and a background color of #fff, even though it's the gold color on the dark background?

- Also on the homepage, in the hero image area with the text "Your Success. Our Guarantee." and the learn more button - both of those are saying there is a foreground color of #fff and a background color of #dddddd. Not sure if that background color can be cheated to be darker, since there is a picture there anyway? If we make it #717171 it should pass the color contrast test.

Thanks!

**Comment by joelworsham** [comment]
For the first point, I can't imagine why this is. My attempted fix was to take the parent item of the text and set the background color to black, even though the image covers that up.

For the second, similar to the first in that the issue really isn't an issue, but the checker doesn't account for the layout. Per your suggestion, I set the background to #717171, which the image will load over.

**Comment by banksashleym** [comment]
Thanks! The hero image text "Your Success. Our Guarantee" passes now, however the slick dots now have color contrast issues as well as the two right column "profile" boxes. 

For the slick dots, it's showing a foreground color of #0a0a0a and the new background of #717171. Could we change the foreground color to #fff?
![1](https://user-images.githubusercontent.com/15264315/36267607-522d88f0-1242-11e8-9e8b-4e9bc12a1011.JPG)

and for the right profile boxes, it's showing a foreground color of #fed05d and a background of #ffffff. Can we match the styling of the right "profile" boxes to that of the "News" boxes? They both look the same to me when I hover, but the middle boxes are not throwing out an error for some reason. Let me know your thoughts.
![2](https://user-images.githubusercontent.com/15264315/36268137-c481eb3e-1243-11e8-9a7d-7f580f3f0ce3.JPG)

**Comment by banksashleym** [comment]
Hi Joel,
This fixed the profile image contrast, but unfortunately the slick dots still don't pass...it's still showing a foreground color of #0a0a0a and a background color of #717171. 

Thanks!

**Comment by joelworsham** [comment]
Hmmm that is quite odd... I'll check this out.
